Client/reasn-client/apps/web/app/Nav.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styles from './Nav.module.css';
+
+
+type Link = {
+    label: string
+    href: string
+};
+
+const Nav = () => {
+return(
+<body>
+  <header>
+    <nav
+      className="
+        flex flex-wrap
+        items-center
+        justify-between
+        w-full
+        h-20
+        text-lg text-white
+        bg-black
+        rounded-b-2xl
+      "
+    >
+    <div>
+        <a className="block text-white font-bold text-3xl mx-60">Reasn.</a>
+    </div>
+
+    <div className="flex space-x-4 items-center mx-60"> 
+      <a className="block text-white mr-8">logowanie</a>   
```

Comment:
```suggestion
      <a>logowanie</a>   
```
Why making it as block? You are using flex in the parent, so block will not affect it. Also - it inherits the text-white, and for mr-8 - it's better to make the parent where flex is defined, gap between 'flexed' items.

Reply:
Deleted everything from child logowanie and added gaps in the parent.